### DISCUSSION TITLE
Possible to keep muted while recommending in vscode

### DISCUSSION THREAD

**Discussion by wu5bocheng** [community/community#48861] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I wanted to take a moment to express my sincere appreciation for the incredible efficiency boost that your product has brought to my coding workflow. Copilot has been an invaluable tool for generating high-quality code and helping me tackle complex programming challenges.

That being said, I did have one suggestion that I hope you might consider implementing. I've noticed that in the VS Code plugin, there is a notification sound that plays every time Copilot recommends a piece of code. While I understand the value of these notifications, they can be quite frequent and disruptive, especially when working in a quiet environment.

As such, I was wondering if there might be a way to allow users to mute or disable these notification sounds, while still benefiting from the powerful code recommendations that Copilot provides. This would help ensure that the product remains user-friendly and accessible for all types of coding environments.

Thank you again for your hard work and dedication to creating such an innovative and helpful tool. I look forward to seeing how Copilot continues to evolve and improve in the future.

**Comment by davecheney** [comment, marked as answer]
This is not normal behavior.  Please contact support and they will escalate to the engineering teams if required.
